### ISSUE TITLE
cp: resources/native-deps: No such file or directory

### ISSUE THREAD

**Issue by viesti** [FieryCod/holy-lambda#30]
Hi! Thanks this neat project :)

I created a new project with `lein new holy-lambda hello-world` and ran `make`, but ended up with the following after a native-image build:

```shell
...
[output:25]        write:   4,450.84 ms,  3.73 GB
# Printing build artifacts to: output.build_artifacts.txt
[output:25]      [total]: 121,535.17 ms,  3.73 GB
cp: resources/native-deps: No such file or directory
make: *** [latest.zip] Error 1
```

I guess the Leiningen template should also create the `resources/native-deps` directory, even though it can be empty?

**Comment by FieryCod** [comment]
Hi @viesti. :)  Thank you so much for trying it out and sorry for this shameful issue.

Provided a fix in version 0.1.19 :) There was also an additional step during native image generation because `jsonista` is not reflection-free. Please retest with the new version.

PS: I'm highly encouraging you to visit our slack channel #holy-lambda if you would like to discuss some stuff :)

**Comment by viesti** [comment]
Oh vau, thank you! Heading onto the channel now :)
